src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpChannelImpl.java

```diff
@@ -1099,7 +1081,7 @@ static native int send0(int fd, long address, int length,
     private static void loadSctpLibrary() {
         IOUtil.load();   /* loads nio & net native libraries */
         java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction<Void>() {
+            new java.security.PrivilegedAction<>() {
```

Comment:
Is it possible to use `import java.security.PrivilegedAction;` instead of fully-qualified name?